clients/drcachesim/tracer/func_trace.cpp

```diff
@@ -227,56 +229,80 @@ instru_funcs_module_load(void *drcontext, const module_data_t *mod, bool loaded)
     uint64 ms_start = dr_get_milliseconds();
     const char *mod_name = get_module_basename(mod);
     NOTIFY(2, "instru_funcs_module_load for %s\n", mod_name);
+    // We need to go through all the functions to identify duplicates and adjust
+    // arg counts before we can write to funclist.  We use this vector to remember
+    // what to write.  We expect the common case to be zero entries.
```

Comment:
The for loop is probably going to fail to find a target function on a typical app library.  The logic is that if a typical big app has at least a dozen libraries, probably only 2 or 3 actually have functions being traced like malloc.  I added " since the average app library probably has zero traced functions in it.".